.npmrc

```diff
@@ -1,2 +1,3 @@
 CYPRESS_INSTALL_BINARY=0
 puppeteer_skip_chromium_download=true
+legacy-peer-deps=true
```

Comment:
Why do we need this?

Reply:
I thought we needed this to manage peer dependency conflicts / backwards compatibility because we're now forcing people to use npm 8+...but it looks like the build works fine without it. Removing.